src/components/NewComps/BookFreeDemoForm/BookFreeDemoForm.module.scss

```
@@ -2,46 +2,130 @@
+.bannerActionBtn {
+  height: 48px;
+  border-radius: 10px !important;
+  min-width: 88px;
 }
 
-.bookdemo_btn_cont {
-  position: fixed;
-  bottom: 10px;
-  right: 0;
+.bannerPhoneIti {
+  :global(.iti) {
```

Stylelint (17.14.1)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)